src/main/interface/msp.c

```diff
@@ -1403,9 +1403,9 @@ bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0);
 #endif
         sbufWriteU8(dst, 0);
-        sbufWriteU16(dst, currentPidProfile->pid[PID_ROLL].F);
-        sbufWriteU16(dst, currentPidProfile->pid[PID_PITCH].F);
-        sbufWriteU16(dst, currentPidProfile->pid[PID_YAW].F);
+        sbufWriteU16(dst, 0);
```

Comment:
wasnt this wrong like in another occurance?

Reply:
I don't think it is, but i'll go and ask loutwice who is way better at the MSP to verify that it is working as expected.

Reply:
i guess he pushed another commit to fix something similar

Reply:
Loutwice has seen this and it seems fine.